kyo-prelude/shared/src/main/scala/kyo2/Local.scala

```diff
@@ -9,14 +9,12 @@ abstract class Local[T]:
 
     val default: T
 
-    private val _get: T < Any =
```

Comment:
The idea here was to cache the `get` operation to avoid allocations but I think it's best to ensure the tracing produces correct information. We can revisit later in case we optimize this code.